overlord/ifacestate/helpers.go

```diff
@@ -820,3 +820,14 @@ func getHotplugSlots(st *state.State) (map[string]*HotplugSlotInfo, error) {
 func setHotplugSlots(st *state.State, slots map[string]*HotplugSlotInfo) {
 	st.Set("hotplug-slots", slots)
 }
+
+func findConnsForHotplugKey(conns map[string]connState, ifaceName, hotplugKey string) []string {
```

Comment:
Should the signature be `map[string]*connState` to avoid copying connState around?
Should we sort the result or indicate that ordering is not deterministic? - I would go for `sort.Strings` since the result will likely be very short.

Reply:
Will address this in a followup due to changes required in unrelated code (getConns, setConns).